test/1.4/lib/T_signal_templates.py

```diff
@@ -53,12 +53,14 @@ def signal_lower(self):
 obj.outsig = stub
 stest.expect_equal(stub.level, 1)
 
-# reverse execution doesn't cause signal pulsing
-transitions = stub.transitions
-simics.SIM_run_command('enable-reverse-execution')
-simics.SIM_run_command('continue 1000')
-simics.SIM_run_command('reverse')
-stest.expect_equal(stub.transitions, transitions)
+# Reverse execution is removed in Simics 7
+if 'rev-execution' in simics.SIM_get_all_classes():
+    # reverse execution doesn't cause signal pulsing
+    transitions = stub.transitions
+    simics.SIM_run_command('enable-reverse-execution')
+    simics.SIM_run_command('continue 1000')
+    simics.SIM_run_command('reverse')
+    stest.expect_equal(stub.transitions, transitions)
```

Comment:
Better to rewrite the test to use the 7 equivalent of revexec (snapshots, right?)

FWIW, I think the purpose of the test is to exercise the following logic in `signal_connect.set`, in `1.4/utility.dml`:
```
        local bool hotplug = SIM_object_is_configured(dev.obj)
            && !SIM_is_restoring_state(dev.obj);
```

Reply:
I agree, but as snapshots API is not finalized yet I think we can wait for that. Posted https://jira.devtools.intel.com/browse/SIMICS-21789 to fix this.